src/components/UserProfile/CavaticaTokenInput.js

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { compose, withState } from 'recompose';
+
+import step2Screenshot from 'assets/cavaticaTokenScreenshot.png';
+import { deleteSecret, setSecret } from 'services/secrets';
+import { CAVATICA } from 'common/constants';
+import { getUser as getCavaticaUser } from 'services/cavatica';
+import { ModalFooter } from 'components/Modal/index.js';
+
+import { css } from 'emotion';
+import { injectState } from 'freactal';
+import RightArrows from 'react-icons/lib/fa/angle-double-right';
+
+const styles = css`
+  span.numberBullet {
+    color: white;
+    background: #00afed;
+    width: 14px;
+    border-radius: 50%;
+    margin: 20px;
+    padding: 10px;
+    height: 14px;
+    display: inline-block;
+    text-align: center;
+  }
+
+  .tokenTitle {
+    padding: 20px;
+    color: #2b388f;
+    font-size: 18px;
+    font-weight: bold;
+  }
+
+  .tokenInput {
+    margin: 20px;
+    padding: 3px;
+    font-size: 16px;
+    border-radius: 10px;
+  }
+`;
+
+const enhance = compose(
+  injectState,
+  withState('cavaticaKey', 'setCavaticaKey', ''),
+  withState('invalidToken', 'setInvalidToken', false),
+);
+
+const submitCavaticaToken = async ({ token, setIntegrationToken, onSuccess, onFail }) => {
+  await setSecret({ service: CAVATICA, secret: token });
+  await getCavaticaUser(token)
+    .then(userData => {
+      setIntegrationToken(CAVATICA, userData);
+      onSuccess();
+    })
+    .catch(response => {
+      setIntegrationToken(CAVATICA, null);
+      deleteSecret({ service: CAVATICA });
+      onFail();
+    });
+};
+
+const CavaticaTokenInput = ({
+  state,
+  effects,
+  theme,
+  cavaticaKey,
+  setCavaticaKey,
+  gen3Key,
+  setGen3Key,
+  editingCavitca,
+  setEditingCavatica,
+  invalidValue,
+  setInvalidToken,
+  ...props
+}) => {
+  return (
+    <div css={styles}>
+      <div>
+        <div>
+          <span className="numberBullet">1</span>
```

Comment:
bubble numbers are a bit mis aligned, try this css

```
    display: flex;
    justify-content: center;
    align-items: center;
```

instead of 

```
    display: inline-block;
    text-align: center;
```

Reply:
actually this is an ordered list, maybe you can switch to <ol>s and <li>s and use this css https://stackoverflow.com/a/23610216/962292